projects/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.component.html

```diff
@@ -1,19 +1,24 @@
 <ng-container
   *ngIf="merchandisingCarouselModel$ | async as merchandisingCarouselModel"
 >
-  <div
-    class="data-cx-merchandising-carousel"
-    [cxAttributes]="merchandisingCarouselModel.metadata"
-    [cxAttributesNamePrefix]="'data-cx-merchandising-carousel'"
-  ></div>
-  <cx-carousel
-    [ngStyle]="carouselStyle$ | async"
-    [items]="merchandisingCarouselModel.items$"
-    [title]="title$ | async"
-    [template]="carouselItem"
-    itemWidth="285px"
-  >
-  </cx-carousel>
+  <ng-container *ngIf="backgroundColor$ | async as backgroundColor">
```

Comment:
Yes, let's avoid all these containers and inline method calls. We can `tap` into the `merchandisingCarouselModel$` in the component logic and set the css variables right away.

Reply:
Done